app/view/LayerTree.js

```diff
@@ -158,6 +158,16 @@ Ext.define('CpsiMapview.view.LayerTree', {
             });
             me.setStore(groupedLayerTreeStore);
 
+            me.getRootNode().cascade(function (node) {
+                var data = node.getData();
+                if (data.leaf) {
+                    var isBaseLayer =  data.get('isBaseLayer');
+                    if (isBaseLayer) {
```

Comment:
```suggestion
                    if (data.leaf && data.get('isBaseLayer')) {
```
surely not blocking

Reply:
What I wanted to say is to merge the 2 if statements to one.

Reply:
done in https://github.com/compassinformatics/cpsi-mapview/pull/189/commits/f7daaa60823018db884ce2a4059da2b4903d2fd2